src/managed/CommonManaged.props

```diff
@@ -19,17 +15,6 @@
     <PackageThirdPartyNoticesFile>$(RepoRoot)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
   </PropertyGroup>
 
-  <!--
-  Once https://github.com/dotnet/buildtools/issues/1548 is fixed, the following version sections should be simply
-  <Import Condition="Exists('$(ToolsDir)BuildVersion.targets')" Project="$(ToolsDir)BuildVersion.targets" />
-  -->
-  <PropertyGroup>
```

Comment:
Is this now handled via dir.props?

Reply:
Yes!  With this refactoring, all the projects are importing the root `dir.props`, like they should be.